src/rest/RequestHandler.js

```diff
@@ -88,8 +88,13 @@ class RequestHandler {
     try {
       res = await request.make();
     } catch (error) {
-      // NodeFetch error expected for all "operational" errors, such as 500 status code
-      throw new HTTPError(error.message, error.constructor.name, error.status, request.method, request.path);
+      // Retry the specified number of times for request abortions
```

Comment:
this should explicitly check that the error is an AbortError (`error.name === 'AbortError'`), and throw it directly otherwise

Reply:
Replied in #library-discussion, only error this can throw is `AbortError`, according to [MDN's docs](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch). The other (`TypeError`) is discarded because we do not build a URL string with user credentials.